packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -267,11 +268,27 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
           rendering: componentRendering,
         };
 
-        const rendered = React.createElement<{ [attr: string]: unknown }>(
+        let rendered = React.createElement<{ [attr: string]: unknown }>(
           component as React.ComponentType,
           this.props.modifyComponentProps ? this.props.modifyComponentProps(finalProps) : finalProps
         );
 
+        const type = rendered.props.type === 'text/sitecore' ? rendered.props.type : '';
+        if (!isEmpty) {
+          rendered = (
+            <ErrorBoundary
+              key={rendered.type + '-' + index}
+              errorComponent={this.props.errorComponent}
+              componentLoadingMessage={this.props.componentLoadingMessage}
+              type={type}
+              isDynamic={(component as JssComponentType).isDynamic}
+              {...rendered.props}
+            >
+              {rendered}
+            </ErrorBoundary>
+          );
+        }
+
```

Comment:
with this refactor does this mean `PlaceholderMetadata` component would not be included in the `ErrorBoundary`?

Reply:
Yes. The main goal of ErrorBoundary is to alert to problems with customer's and app's components. With this change the focus is moved on these components specifically and our built-in stuff will not affect error boundaries.

Reply:
but what if an error occurs in the `PlaceholderMetadata` component?

Reply:
Then we have a larger problem on our side and each page of the site stops working in editing mode.
Which would be roughly the same behavior if PlaceholderMetadata errors are handled by ErrorBoundary - the content will not be shown.